README.md

```diff
@@ -162,6 +162,17 @@ If a field used to be included in streamed entity event data for a particular ta
 
 This may be useful if a field has been deleted from a table in the database, but you wish to continue to analyse past versions of that table which do contain that field, or changes between past versions of that table. The field will contain a ```NULL``` value for the latest version of the table.
 
+### Monitoring and correcting incomplete entity data received from dfe-analytics
+On rare occasions an entity in the application database may be created, updated or deleted, but no event streamed to your events table. This will result in missing or out of date data in tables generated by dfe-analytics.
+
+To detect incomplete or out of date tables, follow dfe-analytics documentation to enable nightly checksums to be streamed to your events table and set the ```compareChecksums: true``` parameter when you call ```dfeAnalyticsDataform()```. If the entity event stream in BigQuery appears to be missing, then the checksum in the event will not match the checksum calculated from the entity event stream in BigQuery. This will cause the ```foo_entity_ids_do_not_match``` assertion generated by dfe-analytics-dataform to fail.
```

Comment:
Could possible say where the parameters are located

Reply:
I've specified now that they go in the JSON passed to the function